elastic/kibana
# [Infrastructure UI] Hosts view: Add links to apm and uptime for a single host

Closes #150985 

## Summary

This PR adds links to APM Traces and Uptime.
<img width="1909" alt="image" src="https://user-images.githubusercontent.com/14139027/229580612-a3d0c03a-4f68-4b09-b333-ffa79c8454e6.png">

# Testing
1. Open Hosts view 
2. Click on the button to open the flyout for a single host
   - Click on the APM traces link and verify the query parameters are correct
   - Click on the uptime link and verify the query parameters are correct
   

https://user-images.githubusercontent.com/14139027/229581672-3c50ea55-e834-4431-aac6-3ed3ff9f96cc.mov
